grails-app/controllers/org/broadinstitute/orsp/ConsentGroupController.groovy

```diff
@@ -183,7 +183,11 @@ class ConsentGroupController extends AuthenticatedController {
     @Override
     show() {
         Issue issue = queryService.findByKey(params.id)
-        if (issueIsForbidden(issue)) {
+        Boolean isUserReporter = false
+        if (params.projectKey != null) {
+            isUserReporter = queryService.findByKey(params.projectKey).reporter == getUser().userName
```

Comment:
Why make the same query twice? On line 185, you already have the issue. This should work:
```
Boolean isUserReporter = issue?.reporter == getUser().userName
```

Reply:
Yes but there are not the same issue. If the user is trying to get into a CG from a Project I need to know if he/she has access. I need not only the id of the CG but also the id of the Project to know if the user is the reporter. One query for the Project and the other one for the CG.

Reply:
Ah ... yes, I missed that. You're querying on two things, the consent group which is `params.id` and then the project, which is `params.projectKey`. So, my feedback here is not accurate, thank you!

One more detail, `queryService.findByKey(params.projectKey).reporter` could be safer if you did this: `queryService.findByKey(params.projectKey)?.reporter`